GeneralsMD/Code/GameEngineDevice/Source/W3DDevice/GameClient/Water/W3DWater.cpp

```diff
@@ -2998,14 +3000,15 @@ void WaterRenderObjClass::setupFlatWaterShader(void)
 
 		Matrix4x4 curView;
 		DX8Wrapper::_Get_DX8_Transform(D3DTS_VIEW, curView);
```

Comment:
Maybe add another function to get D3DXMATRIX here instead of doing this conversion?